src/constants.ts

```diff
@@ -22,10 +22,11 @@ export const SHORT_NAME_RESERVATION_UNLOCK_TIMESTAMP = 1704092400000; // January
 export const MAX_ALLOWED_EVOLUTION_DELAY = 720 * 30;
 export const MINIMUM_ALLOWED_EVOLUTION_DELAY = 3; // 3 blocks for testing purposes, but should be 720 * 7; // 720 blocks per day times 7 days
 export const MINIMUM_ALLOWED_NAME_LENGTH = 5; // names less than 5 characters are reserved for auction
-export const UNDERNAMES_COUNT = 10;
 export const RARITY_MULTIPLIER_HALVENING = 25;
+export const PERMABUY_LEASE_FEE_LENGTH = 10;
 export const ALLOWED_ACTIVE_TIERS = [1, 2, 3];
 export const ANNUAL_PERCENTAGE_FEE = 0.1; // 10% of cost of name
+export const UNDERNAMES_COUNT = 10;
```

Comment:
Maybe call this like MINIMUM_UNDERNAMES or something? or DEFAULT_UNDERNAME_COUNT?

Reply:
added